.gitignore

```diff
@@ -51,3 +51,7 @@ rsconnect/
 inst/doc
 /doc/
 /Meta/
+
+# Pre-compiled stan files
+*.o
```

Comment:
Nice! 

Good to have here. You might also add them to your global `.gitignore` file, that applies to all projects on your machine